From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ViKing <viking.yfy@qq.com>
Date: Mon, 31 Aug 2026 14:56:21 +0800
Subject: [PATCH] net: qualcomm: add PPE firmware marker

Print one build marker after the generic qcom_ppe probe completes.
This patch is the only firmware marker and is refreshed in place for later
images.

Signed-off-by: ViKing <viking.yfy@qq.com>
---
 drivers/net/ethernet/qualcomm/ppe/ppe.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/ethernet/qualcomm/ppe/ppe.c
+++ b/drivers/net/ethernet/qualcomm/ppe/ppe.c
@@ -253,6 +253,7 @@ static int qcom_ppe_probe(struct platfor
 
 	ppe_debugfs_setup(ppe_dev);
 	platform_set_drvdata(pdev, ppe_dev);
+	dev_info_once(dev, "PPE OFFLOAD ViKing/r1 load successful\n");
 
 	return 0;
 }
